common/src/truffle.rs

```diff
@@ -81,7 +81,14 @@ pub struct Documentation {
     /// Contract documentation
     pub details: Option<String>,
     /// Contract method documentation.
-    pub methods: HashMap<String, String>,
+    pub methods: HashMap<String, DocEntry>,
+}
+
+#[derive(Clone, Debug, Default, Deserialize)]
+/// A documentation entry.
+pub struct DocEntry {
```

Comment:
Is this used?

Reply:
Yup!
https://github.com/gnosis/ethcontract-rs/blob/9fd4f359c110928774ad968620d0e27641f07651/derive/src/lib.rs#L313